pkg/daemon/fake_writer.go

```diff
@@ -0,0 +1,46 @@
+package daemon
+
+import (
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	corelisterv1 "k8s.io/client-go/listers/core/v1"
+)
+
+// fakeNodeWriter is a fake single writer to Kubernetes to prevent race conditions
+type fakeNodeWriter struct {
+}
+
+// newFakeNodeWriter Create a new NodeWriter
+func newFakeNodeWriter() NodeWriter {
+	return &fakeNodeWriter{}
+}
+
+// Run reads from the writer channel and sets the node annotation. It will
```

Comment:
super tiny nit: Run**s** ... and sets .. 

also there is some capitalized **S**ets in the comments below which should be lower case.

Reply:
Run is the function name (it's the subject in the sentence)

changed the uppercase->lowercase Sets